### ISSUE TITLE
Bug report- The secret id is not not a valid URI

### ISSUE THREAD

**Issue by leethanh2112** [aztfmod/terraform-azurerm-caf#1742]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Version of the module you are using

version affected 5.6.10 and above

### Rover Version

aztfmod/rover:1.4.6-2306.1405

### Terraform Version

1.4.6

### Affected Resource(s)/Data Source(s)

azurerm_key_vault_certificate

### Terraform Configuration Files

```hcl
keyvault_certificate_issuers = {
  kv_cert_issuer_0 = {
    issuer_name        = "GlobalSign"
    provider_name      = "GlobalSign"
    account_id         = "#{gs-issuer-id}#"
    resource_group_key = "ak_shared_rg"
    keyvault_key       = "ak_shared_kv"
    cert_password_key  = "gs-issuer-password"
    cert_secret_name   = "gs-issuer-password"

    admin_settings = {
      admin1 = {
        first_name    = "ABC"
        last_name     = "XYZ
        email_address = "#{gs-issuer-email-admin}#"
        phone_number  = "#{gs-issuer-phone}#"
      }
    }
  }
```

### Expected Behaviour

The secret id is valid and the keyvault certificate should be create successfully

### Actual Behaviour

Error Message: ERROR: '/subscriptions/0d942dd6-0ff2-4601-a9da-0ae82683948b/resourceGroups/ak_shared_rg/providers/Microsoft.KeyVault/vaults/aksharedkvsecrets/gs-issuer-password' is not not a valid URI

**Comment by khaman-pet** [comment]
@arnaudlh  @iriahk89 help to review the PR. I have tested it from my side. It is working.

**Comment by khaman-pet** [comment]
@arnaudlh @iriahk89 can this be put into the 5.7.7 release?

**Comment by arnaudlh** [comment]
hi @khaman-pet it is merged in main and will be in 5.7.7, let me update tagging accordinly in the issue as well!

**Comment by arnaudlh** [comment]
Closing as merged and included in 5.7.7
